third_party/mbedtls/mbedtls-config.h

```diff
@@ -63,6 +64,7 @@
 #define MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES
 #define MBEDTLS_NO_PLATFORM_ENTROPY
 #define MBEDTLS_PK_C
+#define MBEDTLS_PK_HAVE_ECC_KEYS
```

Comment:
Don't define derived symbol manually. It's available after including any `mbedtls/*.h` public header. You can include `mbedtls/build_info.h` to get all configuration settings (including derived ones) but no function declaration.

Applies to `MBEDTLS_PK_HAVE_ECC_KEYS`, `MBEDTLS_CAN_ECDH` and `MBEDTLS_PK_CAN_ECDSA_SIGN`.